tokio-trace/tokio-trace-core/src/subscriber.rs

```diff
@@ -158,9 +158,30 @@ pub trait Subscriber: 'static {
 
     /// Record a set of values on a span.
     ///
+    /// This method will be invoked when value is recorded on a span.
+    /// Recording multiple values for the same field is possible,
+    /// but the actual behaviour is implementation defined by the subscriber.
```

Comment:
I would reword this as `but the actual behavior is an implementation detail of the subscriber.`